MeesGame/UI/Controls/EditorControls/ColorPicker.cs

```diff
@@ -10,26 +13,18 @@ namespace MeesGame
     class ColorPicker : UIComponent
     {
         private const int width = 200;
-        private const int height = 200;
-        private const int colorsPerRow = 4;
+        private const int height = 250;
```

Comment:
@nessemis I changed the height, I think this changes the height of the entire colorpicker too, so than the comment above should be updated, shouldn't it??